src/pages/miyeon/Login/Login.js

```diff
@@ -1,9 +1,105 @@
 import React from 'react';
+import { Link, withRouter } from 'react-router-dom';
+import './Login.scss';
 
 class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      idVal: '',
+      pwVal: '',
+      buttonColor: false,
+    };
+  }
+
+  handleIdInput = e => {
+    this.setState({
+      idVal: e.target.value,
+    });
+  };
+
+  handlePwInput = e => {
+    this.setState({
+      pwVal: e.target.value,
+    });
+  };
+
+  buttonColorChange = () => {
+    const { idVal, pwVal } = this.state;
+    idVal.includes('@') && pwVal.length >= 5
+      ? this.setState({ buttonColor: true })
+      : this.setState({ buttonColor: false });
+  };
```

Comment:
```suggestion
  buttonColorChange = () => {
    const { idValue, pwValue } = this.state;
    
    const isButtonActive = idValue.includes('@') && pwValue.length >= 5
    
     this.setState({ isButtonActive })
  };
```
- buttonColor보다는(세부 구현, how) isButtonActive(무엇, what)을 표현하는게 좋을거 같아요
- 조건에 이름 붙여 변수로 만들어보셔도 좋을듯 합니다